src/PrivacyPolicy/PrivacyPolicy.jsx

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import showdown from 'showdown';
+import Spinner from '../components/Spinner';
+import './PrivacyPolicy.less';
+
+class PrivacyPolicy extends React.Component {
+  componentDidMount() {
+    this.props.dispatch(dispatch => fetch('/src/privacy_policy.md').then(
```

Comment:
what if there isn't a privacy policy here? wondering if this can be a var vs a hardcoded path

Reply:
also think the dispatch stuff should live in the Redux component vs. the UI component

Reply:
this path is determined by the thing in `customize.sh` that copies it into the `src` directory, so it seems like this would have to be hardcoded